Source/Core/DolphinWX/Debugger/CodeView.cpp

```diff
@@ -298,6 +302,23 @@ void CCodeView::OnPopupMenu(wxCommandEvent& event)
     Refresh();
     break;
 
+  case IDM_ASSEMBLE:
+  {
+    if (!PowerPC::HostIsRAMAddress(m_selection))
+      break;
+    AssemblerEntryDialog dialog(m_selection, this, _("Enter instruction code:"));
+    if (dialog.ShowModal() == wxID_OK)
+    {
+      unsigned long code;
+      if (dialog.GetValue().ToULong(&code, 0) && code <= std::numeric_limits<u32>::max())
+      {
+        m_debugger->InsertBLR(m_selection, code);
```

Comment:
This should probably not be called insert BLR if it's inserting more than just BLRs.